src/com/lilithsthrone/game/dialogue/utils/UtilText.java

```diff
@@ -5233,10 +5233,10 @@ public String parse(String command, String arguments, String target) {
 		}
 	}
 
-	private static GameCharacter character;
```

Comment:
I know this was necessary for what you did, but thanks for making this global var disappear. In my opinion LT has way too many globals.